src/api/app/views/layouts/webui/webui.html.haml

```diff
@@ -31,7 +31,7 @@
     = javascript_tag do
       = yield :head_javascript
       var _paq = _paq || [];
-      $(function() { #{yield :ready_function} });
+      $(function() { #{yield :ready_function} window.onresize = applySmartOverflow; });
```

Comment:
@danidoni I would only add this to the views where it is required

Reply:
Fixed